openstf/metrics/reporter.py

```diff
@@ -91,39 +91,4 @@ def _make_data_series_figures(self, model: RegressorMixin) -> dict:
         }
 
     def _make_feature_importance_figure(self, model: RegressorMixin) -> None:
```

Comment:
Since this function now is only a single line, and the function is 'private' (due to it starting with an underscore), perhaps you can get rid of it altogether?